### DISCUSSION TITLE
Docs saying -o to select command is ok, but it looks like only long form implemented

### DISCUSSION THREAD

**Discussion by vo-va** [umputun/spot#234] [Q&A] [answered]
<img width="1064" alt="image" src="https://github.com/user-attachments/assets/3f27268f-0d12-4386-8da0-160abe9c6091">

https://github.com/umputun/spot/blob/3c62f5351b69ac2bd11546a3007cb8a43ef7762a/cmd/spot/main.go#L53

Hi, sorry for bringing up such a small issue. I am just experimenting with the tool, and I noticed minor things from the documentation aren’t working. I’m reporting it in case it’s a bug or typo.

**Comment by umputun** [comment, marked as answer]
yep, error in docs. feel free to submit a PR with a correction. Alternatively you may add `-o` to cli short form
